package-lock.json

```diff
@@ -0,0 +1,5376 @@
+{
```

Comment:
Please remove this file, this project uses yarn.

Reply:
removed